src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
+import {faCheck} from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.css']
+})
+export class FooterComponent implements OnInit {
+
+  faCheck = faCheck;
```

Comment:
We should use the `library.add(faCheck)` in the module to avoid having to do something like this